### PR TITLE
creationTime should have correct format.

### PR DESCRIPTION
**Pull request by dankmirth**
Context: http://connect-forums.3294226.n2.nabble.com/NwHIN-PD-can-have-invalid-creation-time-td7579571.html#a7579572

This is from the CONNECT 4.2 branch

### CODE REVIEW

**Comment by msweaver** [comment]
OK to test

File: Product/Production/Services/PatientDiscoveryCore/src/test/java/gov/hhs/fha/nhinc/mpi/adapter/component/hl7parsers/HL7DbParser201306Test.java [hunk 1]
```diff
@@ -0,0 +1,18 @@
+package gov.hhs.fha.nhinc.mpi.adapter.component.hl7parsers;
```
Comment (msweaver) [inline Product/Production/Services/PatientDiscoveryCore/src/test/java/gov/hhs/fha/nhinc/mpi/adapter/component/hl7parsers/HL7DbParser201306Test.java:27]: Can you add the connect license to the top of this file?

Comment (dankmirth) [inline Product/Production/Services/PatientDiscoveryCore/src/test/java/gov/hhs/fha/nhinc/mpi/adapter/component/hl7parsers/HL7DbParser201306Test.java:27]: done

**Comment by dankmirth** [comment]
Would it be possible to apply this fix to 4.2?

**Comment by msweaver** [comment]
Yes. Essentially submit the same PR to the 4.2_integration branch. I'll merge them both.

dankmirth notifications@github.com wrote:

Would it be possible to apply this fix to 4.2?

**Comment by dankmirth** [comment]
How should I go about that in a git way?  Should I use cherry picking for
this?  Should I branch the branch and then rebase --onto 4.2_integration?
 I'm not too familiar with git.

On Wed, May 14, 2014 at 3:45 PM, Matthew Weaver notifications@github.comwrote:

> Yes. Essentially submit the same PR to the 4.2_integration branch. I'll
> merge them both.
> 
> dankmirth notifications@github.com wrote:
> 
> Would it be possible to apply this fix to 4.2?
> 
> —
> Reply to this email directly or view it on GitHub<
> https://github.com/CONNECT-Solution/CONNECT/pull/783#issuecomment-43144885>.
> 
> —
> Reply to this email directly or view it on GitHubhttps://github.com/CONNECT-Solution/CONNECT/pull/783#issuecomment-43147510
> .

## 

Thanks,
Dan

## 

CONFIDENTIALITY NOTICE: The information contained in this electronic 
transmission may be confidential. If you are not an intended recipient, be 
aware that any disclosure, copying, distribution or use of the information 
contained in this transmission is prohibited and may be unlawful. If you 
have received this transmission in error, please notify us by email reply 
and then erase it from your computer system.

**Comment by msweaver** [comment]
I think the simplest solution would be to check out 4.2_integration and then cherry pick the commits from the branch you are working on. Give it a shot if it doesn't work out, I can take care of it.

dankmirth notifications@github.com wrote:

How should I go about that in a git way? Should I use cherry picking for
this? Should I branch the branch and then rebase --onto 4.2_integration?
I'm not too familiar with git.

On Wed, May 14, 2014 at 3:45 PM, Matthew Weaver notifications@github.comwrote:

> Yes. Essentially submit the same PR to the 4.2_integration branch. I'll
> merge them both.
> 
> dankmirth notifications@github.com wrote:
> 
> Would it be possible to apply this fix to 4.2?
> 
> —
> Reply to this email directly or view it on GitHub<
> https://github.com/CONNECT-Solution/CONNECT/pull/783#issuecomment-43144885>.
> 
> —
> Reply to this email directly or view it on GitHubhttps://github.com/CONNECT-Solution/CONNECT/pull/783#issuecomment-43147510
> .

## 

Thanks,
Dan

## 

CONFIDENTIALITY NOTICE: The information contained in this electronic
transmission may be confidential. If you are not an intended recipient, be
aware that any disclosure, copying, distribution or use of the information
contained in this transmission is prohibited and may be unlawful. If you
have received this transmission in error, please notify us by email reply
and then erase it from your computer system.

**Comment by msweaver** [comment]
ok to test.

**Pull request merged**